src/api/app/models/event/report_for_package.rb

```diff
@@ -3,6 +3,15 @@ class ReportForPackage < Report
     self.description = 'Report for a package has been created'
     payload_keys :package_name, :project_name
 
+    def self.notification_link_path(notification)
+      return unless Package.exists_by_project_and_name(notification.event_payload['project_name'], notification.event_payload['package_name'])
+
+      Rails.application.routes.url_helpers.package_show_path(package: notification.event_payload['package_name'],
+                                                             project: notification.event_payload['project_name'],
+                                                             notification_id: notification.id,
+                                                             anchor: 'comments-list')
```

Comment:
@eduardoj just saw the `anchor: 'comments-list'` here, this was already there, but seems wrong. Dont makes sense to jump to the comment section for a report on a package. Maybe you can remove it

Reply:
@krauselukas, sure thing. I added another commit to fix that. Thanks!